Add page for decentralized social networks

emmanuel-awosika opened the pull request:
I suggest adding a new page to the use-cases section that covers decentralized social networks built on Ethereum. I think it'll be a great addition since it goes beyond the usual use-cases Ethereum is known for, i.e., DeFi, DAOs, and NFTs.

<!--- Provide a general summary of your changes in the Title above -->

## Description

The article provides an introduction to Ethereum-based social networks, including how they work and why they matter. I included a comparison of decentralized social media networks and traditional social sites to highlight their differences. Finally, the guide reviews popular social media applications like Peepeth and Mirror running on the Ethereum blockchain.


## Related Issue

<!--- This project accepts pull requests related to open issues -->
<!--- If suggesting a new feature or change, please discuss it in an issue first -->
<!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
<!--- Please link to the issue here: -->

minimalsm commented:
Hey @emmanuel-awosika, thanks so much for this 😍

Sorry busy prepping for Devconnect but will try to get this looked at over the next week.

minimalsm commented on src/content/decentralized-social-networks.md:
```suggestion
[MINDS](https://www.minds.com/) is one of the most used decentralized social media platforms. It works like Facebook and has racked up millions of users already.
```

minimalsm replied:
Can you elaborate on 'It works like Facebook'? Not sure what this means.

minimalsm commented on src/content/decentralized-social-networks.md:
_"Users use the platform's native ERC-20 token $MIND to pay for items"_

What items?

minimalsm replied:
"_You can also earn $MIND tokens by publishing popular content, contributing to the ecosystem, and accepting advertising offers._"

We're switching the subject of the paragraph between sentences. e.g.

- _Users can_

- _You can_

minimalsm replied:
"accepting advertising offers."

This might be worth fleshing out.

emmanuel-awosika replied:
> _"Users use the platform's native ERC-20 token $MIND to pay for items"_
> 
> What items?

Meant that users can use the MIND token to pay for extra features (like hiding ads, accessing exclusive content, or earning badges). They can also use it to tip/reward others in the community.

emmanuel-awosika replied:
> "accepting advertising offers."
> 
> This might be worth fleshing out.

It looks like they disabled the peer-to-peer Boost system (I just noticed). Essentially, Boost is a way of getting more impressions on content (like paid ads on Twitter). You could pay someone who has a group to "boost" content in their channel, but I think the feature is controlled by the network itself now. I'll take it out and add another way users can earn.

emmanuel-awosika replied:
> "_You can also earn $MIND tokens by publishing popular content, contributing to the ecosystem, and accepting advertising offers._"
> 
> We're switching the subject of the paragraph between sentences. e.g.
> 
> * _Users can_
> * _You can_

Thanks for pointing this out. I'll edit to reflect the change.

minimalsm commented on src/content/decentralized-social-networks.md:
A small note on writing style: we translate ethereum.org content in over 40 languages, so we try to avoid metaphors like this (i.e. 'warming up') as they don't have the same meaning in different cultures.

```suggestion
Decentralized social media platforms aren't the only ones trying to incorporate blockchain technology; traditional social media are beginning to integrate similar features into their platform.
```

emmanuel-awosika replied:
Got it! Thanks.

minimalsm commented on src/content/decentralized-social-networks.md:
Instagram just announced Polygon NFTs :)

emmanuel-awosika replied:
Awesome! You want me to include it in the content?

minimalsm replied:
If you think it's worthwhile 😀 but it seems like pretty big news that the biggest distributor of jpegs is getting into NFTs 😅

emmanuel-awosika replied:
Definitely! I'll add it to the page.

minimalsm commented on src/content/decentralized-social-networks.md:
We should avoid putting relative date references here as the content is evergreen.

```suggestion
In January 2021, Twitter Blue [rolled out support for NFTs](https://mashable.com/article/twitter-blue-nft-profile-picture), allowing users to connect their wallets and display NFTs as profile pictures. The social media company has also [announced plans](https://www.theverge.com/2021/8/16/22627435/twitter-bluesky-lead-jay-graber-decentralized-social-web) to create a decentralized social network in future.
```

minimalsm commented on src/content/decentralized-social-networks.md:
Should we mention they are working directly with Arbitrum here?

emmanuel-awosika replied:
Yeah, we could do that, if it's necessary (something like "Reddit is working with Arbitrum, a Layer-2 scaling platform for Ethereum"). I felt mentioning the relationship with Ethereum (i.e., using ERC-20 tokens) was enough.

minimalsm commented on src/content/decentralized-social-networks.md:
This isn't social media IMO

emmanuel-awosika replied:
Maybe we can stick to the examples mentioned in the article, then? (Peepeth, Mirror, etc.)

minimalsm commented on src/content/decentralized-social-networks.md:
What is this? A protocol? Their website isn't very clear. Either way, we should focus on dapps as that as what we're describing above—clicking on this will only confuse readers.

```suggestion
```

emmanuel-awosika replied:
I agree on this point. It was initially marketed as some sort of "blockchain-based LinkedIn" where jobseekers can meet companies, but I think they've shifted focus. Now, the business model is offering decentralized code reviews.

minimalsm commented on src/content/decentralized-social-networks.md:
Is this product dead? Looks like Akasha has formed a Foundation but no updates on this project/how to actually use it. As above, if it isn't something users can actually interact with then let's remove it
```suggestion
```

minimalsm commented on src/content/decentralized-social-networks.md:
```diff
@@ -0,0 +1,92 @@
+| Platforms are censorship-resistant and open to everyone.                                                                | Individual access is controlled by centralized entities; platforms can be shut down. 
+| Source code and algorithms are fully available for inspection. Users can vote on platform changes in cases.             | Opaque implementation of algorithms. 
+
+### How do decentralized social networks work? {#decentralized-social-networks-overview}
```
```suggestion
### How do decentralized social networks work? {#decentralized-social-networks-overview}

```

minimalsm commented on src/content/decentralized-social-networks.md:
```diff
@@ -0,0 +1,92 @@
+| Source code and algorithms are fully available for inspection. Users can vote on platform changes in cases.             | Opaque implementation of algorithms. 
+
+### How do decentralized social networks work? {#decentralized-social-networks-overview}
+Decentralized social networks are a class of [decentralized applications (dApps)](https://ethereum.org/en/developers/docs/dapps/)—applications powered by [smart contracts](https://ethereum.org/en/developers/docs/smart-contracts/) deployed on the blockchain. The contract code serves as the backend for these apps and defines their business logic. 
```
```suggestion
Decentralized social networks are a class of [decentralized applications (dApps)](/dapps/)—applications powered by [smart contracts](/smart-contracts/) deployed on the blockchain. The contract code serves as the backend for these apps and defines their business logic. 
```

minimalsm commented on src/content/decentralized-social-networks.md:
```diff
@@ -0,0 +1,92 @@
+sidebarDepth: 2
+summaryPoint1: Blockchain-based platforms for social interaction and content creation and distribution.  
+summaryPoint2: Decentralized social media networks protect user privacy and enhance data security. 
+summaryPoint3: Ethereum's ERC-20 tokens and NFTs are creating new avenues for content monetization. 
```
```suggestion
summaryPoint3: Tokens and NFTs create new ways to monetize content. 
```

minimalsm commented on src/content/decentralized-social-networks.md:
```diff
@@ -0,0 +1,92 @@
+sidebar: true
+sidebarDepth: 2
+summaryPoint1: Blockchain-based platforms for social interaction and content creation and distribution.  
+summaryPoint2: Decentralized social media networks protect user privacy and enhance data security. 
```
_"enhance data security"_

What is the argument for this?

emmanuel-awosika replied:
I was thinking about high-profile data breaches that have affected social media companies in the past. If a decentralized social network integrates with something like Arweave or IPFS, then they can (theorectically) provide more safety for user data.

minimalsm commented on src/content/decentralized-social-networks.md:
```diff
@@ -0,0 +1,92 @@
+| Well-designed monetization framework comprising NFTs, in-app crypto payments, and more.                                 | Poor monetization models.                                            
+| Blockchain security protects users from malicious attacks and safeguards data.                                          | Centralization increases the risk of data breaches and identity theft.                          |
+| Platforms are censorship-resistant and open to everyone.                                                                | Individual access is controlled by centralized entities; platforms can be shut down. 
+| Source code and algorithms are fully available for inspection. Users can vote on platform changes in cases.             | Opaque implementation of algorithms. 
```
This either needs to be removed or explained elsewhere. Open to how you think it could fit in but it might be getting pretty in the weeds trying to explain governance processes here.

```suggestion
| Source code and algorithms are fully available for inspection.           | Opaque implementation of algorithms. 
```

emmanuel-awosika replied:
Definitely agree that we can take this one out.

minimalsm commented on src/content/decentralized-social-networks.md:
Would you consider Mirror to be a social media?

emmanuel-awosika replied:
"Social platform" would be a good way to describe it. I know the term's a bit loose, but I wanted to include a social platform people were familiar with. I think Mirror's a great example if we want to highlight how Ethereum helps user-owned social communities thrive. 

And IMO, Mirror has more going for it as a social platform than, say, Medium. Asides from Twitter or Discord, Mirror is likely the most popular place for Web3-native brands to connect to communities.

minimalsm replied:
Cool 😎 I'm just not very down with the kids in terms of what is/is not considered social media. Agree that Mirror is great to highlight either way.

minimalsm reviewed (changes requested):
Hey @emmanuel-awosika, thank you again for the PR on this 💪 

It is a great start! Left some initial comments to get this closer, a lot of them are open questions.

emmanuel-awosika commented on src/content/decentralized-social-networks.md:
```diff
@@ -0,0 +1,92 @@
+| Decentralized social media                                                                                                      | Traditional social media                                                                 |
+| ----------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------ |
+| Information is stored on the blockchain.                                                                                | Information is stored in servers and data centers.                                              |
+| Global distribution of nodes reduces the possibility of outages.                                                        | Highly susceptible to network failure and service downtimes.                                    |
```
What I had in mind here was to compare the underlying infrastructure and how it affects operations. Specifically, I was thinking of the Facebook/Instagram/WhatsApp outage that happened last year. 

Of course, these outages don't happen very often, but the possibility is higher, since they have less distributed servers. It'll take something like the world's Internet going down or some black swan event to crash Ethereum, so by extension, platforms built on it can benefit from fault-tolerance. 

Thought I'll share my thought process around the point.

minimalsm replied:
Your reasoning makes sense to me but can we drop the hyperbole 😬?

emmanuel-awosika replied:
Yes, that makes sense. Like I said in another comment, I'll simply focus on highlighting the pros (w/o hyperbole) of decentralized social media without directly pitting them against traditional SM platforms.

emmanuel-awosika commented on src/content/decentralized-social-networks.md:
```diff
@@ -0,0 +1,92 @@
+| Decentralized social media                                                                                                      | Traditional social media                                                                 |
+| ----------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------ |
+| Information is stored on the blockchain.                                                                                | Information is stored in servers and data centers.                                              |
+| Global distribution of nodes reduces the possibility of outages.                                                        | Highly susceptible to network failure and service downtimes.                                    |
```
Sure, they'll definitely have a global distribution of servers. But I think the main difference here is that a blockchain like Ethereum likely has even more servers (i.e., nodes) sustaining the network. Plus, people are _incentivized_ to run these mini-servers--so you can at least bet that some nodes will always be online.

minimalsm replied:
> But I think the main difference here is that a blockchain like Ethereum likely has even more servers (i.e., nodes) sustaining the network.

But we're not going to be storing the majority of data on Ethereum. [Livepeer is a good example of this](https://livepeer.org/). I'd bet Livepeer nodes are definitely fewer in number than YouTube servers. Not strongly opposed to keeping it, but I do think the point itself ignores the reality of the current situation.

emmanuel-awosika replied:
Alright, that's a fair point. Would it be okay to replace the comparison with a line like "As dapps running on Ethereum network, which is sustained by a global, peer-to-peer network of nodes, decentralized social networks are less susceptible to downtime and outages."?

I'm taking a cue from the benefits of dapps listed on the Ethereum.org site: 

_"Once the dapp is live on Ethereum, it will only go down if Ethereum itself goes down. Networks of Ethereum's size are notoriously difficult to attack."_

emmanuel-awosika commented on src/content/decentralized-social-networks.md:
```diff
@@ -0,0 +1,92 @@
+| ----------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------ |
+| Information is stored on the blockchain.                                                                                | Information is stored in servers and data centers.                                              |
+| Global distribution of nodes reduces the possibility of outages.                                                        | Highly susceptible to network failure and service downtimes.                                    |
+| Users can enjoy privacy in communication.                                                                               | Users are forced to exchange privacy for free access.                                          |
```
I was thinking along the lines of decentralized identity here. You properly have to share personally identifiable information (PII) like names, email addresses, residential addresses, etc., if you need to use a regular social media app. Since dApps allow people to use wallets as sign-in methods, they likely have higher privacy standards. 

And yeah, I know E2E is gaining ground, but if the code is never open-source, there's really no way people know if their messages are truly private. I mean, LinkedIn pops up a "Set up meeting" notification when "meeting" is mentioned in my messages. Which sort of gives the notion that messages are still being read (even if it's AI).

minimalsm replied:
> Perhaps we can say "users can enjoy privacy of information vs users exchange privacy for free access" instead?

I think the current phrasing is ok (maybe a little combative?). Perhaps we could soften it a little? e.g. 'Users often have to exchange privacy for free access'

My main problem was it appears to be comparing apples and oranges on the pros vs cons.

emmanuel-awosika replied:
So, are we sticking the benefits listing style for this page?

emmanuel-awosika commented on src/content/decentralized-social-networks.md:
Great point. Thanks.

minimalsm commented on src/content/decentralized-social-networks.md:
```diff
@@ -0,0 +1,91 @@
+summaryPoint2: Decentralized social media networks protect user privacy and enhance data security. 
+summaryPoint3: Tokens and NFTs create new ways to monetize content. 
+---
+Today, social media platforms play a huge role in how we communicate and interact with each other. Social networking services have gone beyond mere fads and now underpin the digital economy. 
+
+However, centralized control of these platforms has created many problems. Data breaches, server outages, deplatforming, censorship, privacy violations—these are some of the trade-offs social media users are forced to accept. 
+
+To combat these issues, developers are building novel social media applications on decentralized networks like Ethereum. Decentralized social networks are poised to fix the the ills of legacy social platforms and improve the overall user experience. 
```
```suggestion
Social networks play a massive role in our daily communications and interactions. However, centralized control of these platforms has created many problems: data breaches, server outages, de-platforming, censorship, and privacy violations are some of the trade-offs social media often make. To combat these issues, developers are building social networks on Ethereum. Decentralized social networks can fix many of the problems of traditional social networking platforms and improve users' overall experience.
```

minimalsm commented on src/content/decentralized-social-networks.md:
```suggestion
### Reddit {#reddit}
```

minimalsm commented on src/content/decentralized-social-networks.md:
```suggestion
### Twitter {#twitter}
```

minimalsm commented on src/content/decentralized-social-networks.md:
```suggestion
### Instagram {#instagram}
```

minimalsm commented on src/content/decentralized-social-networks.md:
- Let's assume users know of Instagram and its popularity
- Slight structure reorg of the last sentence to put the subject first

```suggestion
In May 2022, [Instagram announced support for NFTs](https://about.instagram.com/blog/announcements/instagram-digital-collectibles) on Ethereum and Polygon. Users can post NFTs directly to Instagram by connecting their Ethereum wallet. 
```

minimalsm commented on src/content/decentralized-social-networks.md:
Could we add a short description of what these are? They aren't very appealing for users to click right through as they are.

emmanuel-awosika replied:
Just added the descriptions.

minimalsm commented on src/content/decentralized-social-networks.md:
```suggestion
### MINDS {#minds}
```

minimalsm commented on src/content/decentralized-social-networks.md:
```suggestion
### Mirror {#mirror}
```

minimalsm commented on src/content/decentralized-social-networks.md:
```suggestion
### Peepeth {#peepeth}
```

minimalsm commented on src/content/decentralized-social-networks.md:
```diff
@@ -0,0 +1,89 @@
+template: use-cases
+emoji: ":mega:"
+sidebar: true
+sidebarDepth: 2
+summaryPoint1: Blockchain-based platforms for social interaction and content creation and distribution.  
```
Adding a placeholder image for now.

```suggestion
sidebarDepth: 2
image: ../../assets/ethereum-learn.png
summaryPoint1: Blockchain-based platforms for social interaction and content creation and distribution.  
```

minimalsm commented on src/content/decentralized-social-networks.md:
```diff
@@ -0,0 +1,89 @@
+---
+Social networks play a massive role in our daily communications and interactions. However, centralized control of these platforms has created many problems: data breaches, server outages, de-platforming, censorship, and privacy violations are some of the trade-offs social media often make. To combat these issues, developers are building social networks on Ethereum. Decentralized social networks c ...
+
+## What are decentralized social networks? {#what-are-decentralized-social-networks}
```
```suggestion
## What are decentralized social networks? {#what-are-decentralized-social-networks}

```

minimalsm commented on src/content/decentralized-social-networks.md:
```suggestion
## Benefits of decentralized social media platforms {#benefits-of-decentralized-social-media}
```

minimalsm commented:
Pushed a few commits re: formatting. Going to give it a quick skim after it builds but I think this is ready to ship to the world 🎉

wackerow commented on src/content/decentralized-social-networks.md:
Similar to DID page, if we're going to use the `use-case` template, we could consider adding a bit more here in the same format as the other use case pages. Similar to my comment on that PR, don't think it needs to be a blocker.

wackerow commented on src/content/decentralized-social-networks.md:
```diff
@@ -0,0 +1,94 @@
+summaryPoint1: Blockchain-based platforms for social interaction and content creation and distribution.  
+summaryPoint2: Decentralized social media networks protect user privacy and enhance data security. 
+summaryPoint3: Tokens and NFTs create new ways to monetize content. 
+---
```
```suggestion
---

```

wackerow reviewed (changes requested):
@emmanuel-awosika Thanks for another great PR! I left a handful of small code suggestions that shouldn't be too controversial. 

Couple other thoughts here.. commented about how it would be nice to add a note about disintermediation, aka, removing the middle man, and connecting people directly... really important point that I think we should touch on directly (perhaps as a summary point?)

Also, was curious your take on how DeSoc would fit into all of this. Should this page be DeSoc more broadly?
And, Lens protocol was another thing I feel like would be worth mentioning here. They're building out primitives for decentralized social networking with the plan for anyone to be able to build front ends for these primitives. 
...I don't think these two ideas need to be blockers, but just want to consider them.

Lastly, house cleaning... we need to link out to this. Basic places would be same as my comments on the DID PR, we could add to Use Ethereum menu in nav and footer, and add to the shared "use case" menu dropdown.

(And if we do build in DeSoc in some way, perhaps a redirect for `/desoc/`)

wackerow reviewed (commented):
@minimalsm The relative path for the image here isn't working because this file is in the `/content` folder directly instead of nested in a folder as `/title/index.md`. We just need to rename/move the file to `/content/decentralized-social-networks/index.md`.

emmanuel-awosika commented on src/content/decentralized-social-networks.md:
Added this point to the section.

emmanuel-awosika commented:
> @emmanuel-awosika Thanks for another great PR! I left a handful of small code suggestions that shouldn't be too controversial.
> 
> Couple other thoughts here.. commented about how it would be nice to add a note about disintermediation, aka, removing the middle man, and connecting people directly... really important point that I think we should touch on directly (perhaps as a summary point?)
> 
> Also, was curious your take on how DeSoc would fit into all of this. Should this page be DeSoc more broadly? And, Lens protocol was another thing I feel like would be worth mentioning here. They're building out primitives for decentralized social networking with the plan for anyone to be able to build front ends for these primitives. ...I don't think these two ideas need to be blockers, but just want to consider them.
> 
> Lastly, house cleaning... we need to link out to this. Basic places would be same as my comments on the DID PR, we could add to Use Ethereum menu in nav and footer, and add to the shared "use case" menu dropdown.
> 
> (And if we do build in DeSoc in some way, perhaps a redirect for `/desoc/`)

- I added your point about disintermediation to the "Benefits" section. I think the use-case template allows for only three summary points? 
- Added something about r/CryptoCurrency using Reddit Community Points, so users can see what it looks like in practice. 
- Added Lens Protocol to the "Use decentralized social media section." I changed the section header since Lens Protocol doesn't directly use Ethereum.
- If you mean "DeSoc" as in "Decentralized social media", then I think it's a good choice.

minimalsm commented on src/content/decentralized-social-networks.md:
```suggestion
- **[Status.im](https://status.im/)** - _Status is a secure messaging app that uses an open-source, peer-to-peer protocol, and end-to-end encryption to protect your messages from third parties._ 
- **[Mirror.xyz](https://mirror.xyz/)** - _Mirror is a decentralized, user-owned publishing platform built on Ethereum for users to crowdfund ideas, monetize content, and build high-value communities._ 
- **[Lens Protocol](https://lens.xyz/)** - _Lens Protocol is a composable and decentralized social graph helping creators take ownership of their content wherever they go in the digital garden of the decentralized internet._
```

minimalsm commented on src/content/decentralized-social-networks.md:
```suggestion
### Videos {#videos}

```

minimalsm commented on src/content/decentralized-social-networks.md:
```suggestion
### Communities {#communities}

```

minimalsm commented on src/content/decentralized-social-networks.md:
```suggestion
## Further reading {#further-reading}
```

minimalsm commented on src/content/decentralized-social-networks.md:
```suggestion
- [r/CryptoCurrency subreddit](https://www.reddit.com/r/CryptoCurrency/)
```

minimalsm commented on src/content/social-networks/index.md:
```diff
@@ -0,0 +1,119 @@
+---
+title: Decentralized social media networks
```
```suggestion
title: Decentralized social networks
```

minimalsm commented on src/content/social-networks/index.md:
```diff
@@ -0,0 +1,119 @@
+
+Traditional social media platforms rely on databases to store user information, program code, and other forms of data. But this creates single points-of-failure and introduces significant risk. For instance, Facebook's servers infamously [went offline for hours](https://www.npr.org/2021/10/05/1043211171/facebook-instagram-whatsapp-outage-business-impact) last year, cutting off users from the plat ...
+
+Decentralized social media website exist on a peer-to-peer network comprising thousands of nodes around the globe. Even if some nodes fail, the network will run uninterrupted, making applications resistant to failures and outages. 
```
```suggestion
Decentralized social networks exist on a peer-to-peer network comprising thousands of nodes around the globe. Even if some nodes fail, the network will run uninterrupted, making applications resistant to failures and outages. 
```

minimalsm commented on src/content/social-networks/index.md:
```suggestion
## Benefits of decentralized social networks {#benefits}
```

minimalsm commented on src/content/social-networks/index.md:
```suggestion
1. Decentralized social networks are censorship-resistant and open to everyone. This means users cannot be banned, deplatformed, or restricted arbitrarily. 
```

minimalsm commented on src/content/social-networks/index.md:
```suggestion
## Decentralized social networks on Ethereum {#ethereum-social-networks}
```

minimalsm commented on src/content/social-networks/index.md:
```suggestion
[MINDS](https://www.minds.com/) is one of the most used decentralized social networks. It works like Facebook and has racked up millions of users already.
```

minimalsm commented on src/content/social-networks/index.md:
```suggestion
## Web2 social networks on Ethereum {#web2-social-networks-and-ethereum}
```

minimalsm commented on src/content/social-networks/index.md:
```suggestion
## Use decentralized social networks {#use-decentralized-social-networks}
```

minimalsm commented on src/content/social-networks/index.md:
```suggestion
The Ethereum network has become the preferred tool for developers creating decentralized social media owing to the popularity of its tokens (ERC-20/ERC-721) and its massive user base. Here are some examples of Ethereum-based social networks:
```

minimalsm commented on src/content/social-networks/index.md:
```suggestion
6. Decentralized social networks rely on decentralized storage, not centralized databases, which are considerably better for safeguarding user data.
```

minimalsm commented on src/content/social-networks/index.md:
```diff
@@ -0,0 +1,119 @@
+
+## What are decentralized social networks? {#what-are-decentralized-social-networks}
+
+Decentralized social media networks are blockchain-based platforms that allow users to exchange information as well as publish and distribute content to audiences. Because these applications run on the blockchain, they are capable of being decentralized and resistant to censorship and undue control. 
```
```suggestion
Decentralized social networks are blockchain-based platforms that allow users to exchange information as well as publish and distribute content to audiences. Because these applications run on the blockchain, they are capable of being decentralized and resistant to censorship and undue control. 
```

minimalsm commented on src/content/social-networks/index.md:
```suggestion
Posts published on Mirror are permanently stored on Arweave, a decentralized storage platform, and can be minted as collectable [non-fungible tokens (NFTs)](/nft/) known as Writing NFTs. Writing NFTs are completely free for writers to create, and collection happens on an Ethereum L2 — making transactions inexpensive, fast, and environmentally friendly.
```

minimalsm reviewed (approved):
Awesome work once again @emmanuel-awosika ❤️

- Made a few changes requested from feedback by Mirror team 
- Added it to nav/use cases pages

Merging 🙏🎉

emmanuel-awosika commented:
> Awesome work once again @emmanuel-awosika ❤️
> 
> * Made a few changes requested from feedback by Mirror team
> * Added it to nav/use cases pages
> 
> Merging 🙏🎉

Glad I could help. Thank you for providing useful feedback on earlier drafts, too.
